src/libs/dutil/thmutil.cpp

```diff
@@ -2331,6 +2336,11 @@ static HRESULT ParseControls(
         {
             type = THEME_CONTROL_TYPE_TAB;
         }
+        else if (CSTR_EQUAL == ::CompareStringW(LOCALE_INVARIANT, 0, bstrType, -1, L"Combobox", -1) ||
+                 CSTR_EQUAL == ::CompareStringW(LOCALE_INVARIANT, 0, bstrType, -1, L"cm", 2))
+        {
+            type = THEME_CONTROL_TYPE_COMBOBOX;
+        }
 
         if (THEME_CONTROL_TYPE_UNKNOWN != type)
         {
```

Comment:
Yes, I started writing a wrapper to insert strings and get the current selection, and I realized it didn't have anything interesting to do, the messages CB_ADDSTRING and CB_GETCURSEL are just extremely simple. If you feel I should add wrapper functions for these (that are probably going to be oneliners), speak up, I don't feel strongly about it.